File: mongodb/less/apis/demo/hello/py/get.py
from mongodb_client_py import insert_item, get_all_items
import json

def process(request, response):
    # Insert item.
    insert_item({ 'bar': 'baz' })

    # Get all items.
    result = get_all_items()
    
    # Return HTTP response.
    response.update({
        "body": json.dumps(result),
        "statusCode": 200
    })

    return response

File: mongodb/less/apis/demo/hello/js/get.js
const { insert_item, get_all_items } = require('mongodb_client_js');

module.exports = {
  process: async (request, response) => {
    // Create an item.
    await insert_item({ foo: 'bar '});

    // Fetch all items.
    const result = await get_all_items();

    // Return HTTP response.
    response.statusCode = 200;
    response.body = JSON.stringify(result);
    return response;
  }
}
